partiql-value/src/lib.rs

```diff
@@ -553,13 +554,25 @@ impl Value {
     }
 
     #[inline]
-    pub fn coerce_to_tuple(self) -> Tuple {
+    pub fn coerce_into_tuple(self) -> Tuple {
         match self {
             Value::Tuple(t) => *t,
-            Value::Missing => tuple![],
+            _ => self
+                .into_bindings()
+                .map(|(k, v)| (k.unwrap_or_else(|| "_1".to_string()), v))
+                .collect(),
+        }
+    }
+
+    #[inline]
+    pub fn coerce_to_tuple(&self) -> Tuple {
+        match self {
+            Value::Tuple(t) => t.as_ref().clone(),
             _ => {
-                let fresh_key = "_1"; // TODO don't hard-code 'fresh' keys
-                tuple![(fresh_key, self)]
+                let fresh = "_1".to_string();
+                self.as_bindings()
+                    .map(|(k, v)| (k.unwrap_or(&fresh), v.clone()))
+                    .collect()
```

Comment:
Should `coerce_to_tuple` still remain? The CHANGELOG changes mentioned it was replaced w/ `coerce_into_tuple`.

Reply:
`coerce_into_tuple` takes ownership of `self`; `coerce_to_tuple` takes a reference `&self` and clones only if necessary.